### DISCUSSION TITLE
Sobre Algoritmo de Complejidad O(n*log(n))

### DISCUSSION THREAD

**Discussion by elbosco** [IIC2133-PUC/2024-2#121] [Tarea 1]
Hola,
Se puede aplicar luego de acciones determinadas afuera del condicional del comando, un algoritmo de Complejidad O(n*log(n)) como por ejemplo HeapSort?
Saludos,

**Comment by tomastrivino** [comment]
A qué te refieres con "afuera del condicional del comando"?

**Reply by lfarr1110** [reply]
Me parece que se refiere a si afuera de cualquier comando (ENTER, LEAVE, etc) se puede realizar un algoritmo de una complejidad cualquiera, como que en cada iteracion del for principal se llame a algun tipo de sort (por ejemplo).
